### PR TITLE
[lldb][NFC] Wrap Swift-only code in DWIM-print into commented blocks

### PR DESCRIPTION
**Pull request by augusto2112**

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

File: lldb/source/Commands/CommandObjectDWIMPrint.cpp [hunk 1]
```diff
@@ -142,12 +153,11 @@ bool CommandObjectDWIMPrint::DoExecute(StringRef command,
   //   3. Require addresses to be on the heap
   std::string modified_expr_storage;
   // Either Swift was explicitly specified, or the frame is Swift.
-  bool is_swift = false;
+  bool is_swift = language == lldb::eLanguageTypeSwift;
```
Comment (kastiglione) [inline lldb/source/Commands/CommandObjectDWIMPrint.cpp:156]: this change makes the following if statement redundant.

**Review by kastiglione — APPROVED** [review]
thanks. One comment to address

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by JDevlieghere** [comment]
@swift-ci please test windows

**Pull request merged**